src/backend/api/meals.js

```diff
@@ -1,88 +1,196 @@
-const express = require("express");
+const express = require("express")
 const router = express.Router();
 const knex = require("../database");
 
-router.get("/", async (request, response) => {
-  try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
-    const meal = await knex("meals").select();
-    response.json(meal);
 
-  } catch (error) {
-    throw error;
-  }
-});
+//  /api/meals/:meal_id/reviews	GET	Returns all reviews for a specific meal.
+//  Reviews
 
-///api/meals	POST	Adds a new meal to the database
+router.get("/:meal_id/reviews", async (request, response) => {
 
-router.post('/', async (request, response) => {
```

Comment:
Why are you removing the POST route? I will wait with reviewing this file because it looks things that shouldn't have been deleted has been deleted so the diff looks a bit messy. Can you try to bring back the POST route and clean things up so?